.github/workflows/testing.yml

```diff
@@ -0,0 +1,31 @@
+name: Go Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
I think Windows is still supported, so you should include it here too. Maybe including macOS isn't a bad idea either.